docs/handbook/systems-administration.md

```diff
@@ -81,24 +93,114 @@ su
 Password:
 ```
 
+
 ### SUperuser DO: sudo(1m)
 
-The `sudo` command, i.e., superuser do, permits a user to use _all_ supperuser
-commands without having to become the superuser. A sudo enabled user, simply
-precedes a command with `sudo`.
+To overcome the limitations of providing complete root access, the `sudo` command
+was developed. The sudo command, i.e., superuser do, or substitute user do,
+permits select root access on a user, command or machine level. It effectively
+allows users to execute a number of commands as another user. This other user is
+most frequently root.
+
 
-To enable a user the ability to use `sudo`, the superuser edits `/etc/sudoers`.
-This should ideally be done as follows:
+#### Configuring sudo
+
+To enable a user the ability to use sudo, `/etc/sudoers` must be edited.
+It can be configured such that a user is privileged to use one, several or all
+superuser commands. Syntax errors to this file can cause severe havoc to the
+system. To this end this file should ideally be edited using `visudo`. This
+performs various syntax checks. `sudoers`(4) provides details on the precise means
+to appropriately add a user to use sudo.
+
+visudo will invoke the editor defined in the system environment variable EDITOR,
+and defaults to vi. If you'd prefer to use another editor, for example emacs,
+call visudo as follows: `EDITOR=emacs visudo`
+
+More flexible, however, is assigning one or several commands to a number of
```

Comment:
'group' has such a fixed interpretation that I specifically wanted to avoid using 'group' and its associated Unix connotations, just a collection of users and later form the group:
But this is so small and trivial that I can go with your suggestion.  ok